tools/basicplot/.shed.yml

```diff
@@ -0,0 +1,13 @@
+# .shed.yml supporting automatic pushes.
+owner: chamaelj
```

Comment:
Before pushing the tool to the toolsheds, you will have to create your login/password in these toolshed, then to grant me for write access to your repo, so that I can maintain the tool in the future. I'll show you